pkg/steps/source.go

```diff
@@ -254,7 +255,18 @@ func handleBuild(buildClient BuildClient, build *buildapi.Build, dry bool) error
 			return fmt.Errorf("could not create build %s: %v", build.Name, err)
 		}
 	}
-	return waitForBuild(buildClient, build.Namespace, build.Name)
+	err := waitForBuild(buildClient, build.Namespace, build.Name)
+	if err == nil && len(artifactDir) > 0 {
+		if err:= gatherSuccessfulBuildLog(buildClient, artifactDir, build.Namespace, build.Name); err != nil{
+		// log error but do not fail successful build
+		log.Printf("problem gathering successful build %s logs into artifacts: %v", build.Name, err)
+		}
+		if aerr != nil {
```

Comment:
oop ... accepting @droslean 's commit did not delete the `aerr` block ... will address